app/common/src/index.ts

```diff
@@ -15,26 +15,29 @@
  * URL protocol scheme for deep links to authentication flow pages, without the `:` suffix.
  *
  * For example: the deep link URL
- * `enso://authentication/register?code=...&state=...` uses this scheme.
+ * `enso://authentication/register?code=...&state=...` uses this scheme
  */
-export const DEEP_LINK_SCHEME: string
+export const DEEP_LINK_SCHEME: string = 'enso'
 
 /** Name of the product. */
-export const PRODUCT_NAME: string
+export const PRODUCT_NAME: string = 'Enso'
 
 /** Company name, used as the copyright holder. */
-export const COMPANY_NAME: string
+export const COMPANY_NAME: string = 'New Byte Order sp. z o.o.'
```

Comment:
I don't think that's correct. It should be
```suggestion
export const COMPANY_NAME: string = 'Enso International, Inc.'
```

Reply:
Not sure about this. This PR doesn't actually change that value. I just combined `index.js`/`index.d.ts` into a single `index.ts` file. You can see previous assignment here: https://github.com/enso-org/enso/blob/e5a1c5a6fa42f583521f2f322c0e1197e5491cf6/app/common/src/index.js#L22